### PR TITLE
add more info on mongodb

### PR DESCRIPTION
**Pull request by k-okada**
adbe52f1e (Kei Okada, 10 seconds ago)
    add information based on   https://github.com/jsk-ros-pkg/jsk_robot/issues/1706

af53a852a (Kei Okada, 19 minutes ago)
    add example launch/script for mongo db

### CODE REVIEW

**Review by mqcmd196 — COMMENTED** [review]
I'll check it.
Minor fix
`robot_databse_monogo_example.py` -> `robot_database_mongo_example.py`

**Comment by tkmtnt7000** [comment]
With this change(4212b09fc796138096f96f36259d70751f1043e8), we can get pr1040 or fetch1075 existing smach database. This implementation is temporary, though.

**Comment by knorth55** [comment]
@tkmtnt7000 Super cool!

**Comment by mqcmd196** [comment]
@tkmtnt7000 which branch https://github.com/jsk-ros-pkg/jsk_robot/commit/4212b09fc796138096f96f36259d70751f1043e8 belongs to?

**Comment by tkmtnt7000** [comment]
Maybe https://github.com/tkmtnt7000/jsk_robot/tree/add_mongo_example_73B2

File: jsk_robot_common/jsk_robot_startup/scripts/robot_databse_mongo_example.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+# global variabels
+JST = tz.gettz('Asia/Tokyo')
+bridge = CvBridge()
+
+# sample functions
+def query_latest_smach():
+    try:
+        rospy.loginfo("Loading last smach execution...")
+        last_msg, _ = msg_store.query(
```
Comment (mqcmd196) [inline jsk_robot_common/jsk_robot_startup/scripts/robot_databse_mongo_example.py:38]:
```python
last_msg = msg_store.query(
    SmachContainerStatus._type,
    {"active_states": "INIT"},
    single=True,
    sort_query=[("_meta.inserted_at", pymongo.DESCENDING)]
)
```

File: jsk_robot_common/jsk_robot_startup/scripts/robot_databse_mongo_example.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            SmachContainerStatus._type,
+            {"info": "START"},
+            single=True,
+            sort_query=[("_meta.inserted_at", pymongo.DESCENDING)]
+        )
+        msgs = msg_store.query(
```
Comment (mqcmd196) [inline jsk_robot_common/jsk_robot_startup/scripts/robot_databse_mongo_example.py:44]:
```python
msgs = msg_store.query(
    SmachContainerStatus._type,
    {"header.stamp.secs": {"$gt": last_msg[0].header.stamp.secs}},
    sort_query=[("_meta.inserted_at", pymongo.ASCENDING)]
)
```

File: jsk_robot_common/jsk_robot_startup/scripts/robot_databse_mongo_example.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    rospy.init_node('sample_robot_database')
+    db_name = 'jsk_robot_lifelog'
+    col_name = 'basil' # pr1012, fetch17 etc..
+    msg_store = MessageStoreProxy(database=db_name, collection=col_name)
+
+    print("> sample program for robot database")
+    print("> 1: get latest smach data")
+    print("> 2: get last 1 hours image data")
+    #key = int(input("> {1, 2, 3..} : "))
+    key = 2
```
Comment (mqcmd196) [inline jsk_robot_common/jsk_robot_startup/scripts/robot_databse_mongo_example.py:94]: revert this pls

**Review by mqcmd196 — CHANGES_REQUESTED** [review]

File: jsk_robot_common/jsk_robot_startup/scripts/robot_databse_mongo_example.py (hunk 3 above)
Comment (mqcmd196) [reply jsk_robot_common/jsk_robot_startup/scripts/robot_databse_mongo_example.py:94]: `key = 2`

**Pull request merged**